config/clusters/openscapes/common.values.yaml

```diff
@@ -47,41 +47,17 @@ basehub:
       allowNamedServers: true
       config:
         JupyterHub:
-          authenticator_class: cilogon
-        CILogonOAuthenticator:
-          allowed_idps:
-            http://github.com/login/oauth/authorize:
-              username_derivation:
-                username_claim: "preferred_username"
-        OAuthenticator:
-          # WARNING: Don't use allow_existing_users with config to allow an
-          #          externally managed group of users, such as
-          #          GitHubOAuthenticator.allowed_organizations, as it breaks a
-          #          common expectations for an admin user.
-          #
-          #          The broken expectation is that removing a user from the
-          #          externally managed group implies that the user won't have
-          #          access any more. In practice the user will still have
-          #          access if it had logged in once before, as it then exists
-          #          in JupyterHub's database of users.
-          #
-          allow_existing_users: True
+          authenticator_class: github
+        GitHubOAuthenticator:
+          oauth_callback_url: https://staging.openscapes.2i2c.cloud/hub/oauth_callback
+          allowed_organizations:
+            - 2i2c-org:hub-access-for-2i2c-staff
```

Comment:
With oauthenticator 16, we don't need this - its resolved by:

```yaml
        add_staff_user_ids_to_admin_users: true
        add_staff_user_ids_of_type: "github"
```

```suggestion
```

Reply:
Oh wait... I'll confirm my thoughts first. I made a search to confirm it by it was made incorrectly.

Reply:
Thanks @consideRatio, I'll hold off for now.

Reply:
Thanks @consideRatio. Just to verify, is that true when `OAuthenticator.allow_existing_users` is not set to True as well?

Reply:
https://github.com/2i2c-org/infrastructure/pull/3358 makes us 100% consistent

Reply:
> Thanks @consideRatio. Just to verify, is that true when `OAuthenticator.allow_existing_users` is not set to True as well?

Yes! With Oauthenticator 16 - a user allowed in any way is allowed, the union of ways to allow users as compared to the intersection.

`OAuthenticator.allow_existing_users` is one way to allow users, `admin_users`, `allowed_users`, `allowed_organizations` are other ways as well. This PR looks correct to me with the intent I've understood.

Reply:
Thanks @consideRatio